pawnShop/src/main/java/com/example/pawnShop/ManualMapper/ProductManualMapperImp.java

```diff
@@ -0,0 +1,26 @@
+package com.example.pawnShop.ManualMapper;
+
+import com.example.pawnShop.Dto.Product.ProductTypeDto;
+import com.example.pawnShop.Entity.ProductType;
+import com.example.pawnShop.ManualMapper.Contract.ProductManualMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductManualMapperImp implements ProductManualMapper {
+    @Override
+    public ProductTypeDto mapToProductTypeDto(ProductType productType) {
```

Comment:
this is exactly what the builder should look like ! Bravo! 
But when calling this method, the returned value should be stored in a value with type ;)
don't use 'var' when calling this method, instead declare the type of the returned entity.